src/pages/admin/ModalPopup/index.tsx

```diff
@@ -21,10 +30,41 @@ import {
 
 interface IModalPopup {
   onClose: () => void;
+  userId: string | undefined;
 }
 
-function ModalPopup({ onClose }: IModalPopup) {
+function ModalPopup({ onClose, userId }: IModalPopup) {
   const { t } = useTranslation();
+  const navigate = useNavigate();
+  const { showToast } = useToast();
+
+  const [deleteUserByAdmin] = useDeleteUserByAdminMutation();
+
+  async function handleDeleteUser(id: string | undefined) {
```

Comment:
handleDeleteUser(id?: string)  should work

Reply:
fixed